src/TLuaInterpreter.cpp

```diff
@@ -167,80 +167,80 @@ void TLuaInterpreter::slot_replyFinished(QNetworkReply * reply )
     }
 
     QString localFileName = downloadMap.value( reply );
-    TEvent e;
+    TEvent event;
```

Comment:
I had to choose `mudletEvent` in a couple of places because there was already a Qt `event` in the same code. :laughing: